src/core/time.d

```diff
@@ -1393,18 +1393,7 @@ public:
     }
 
 
-    /++
-        $(RED Deprecated. Please use $(LREF split) instead. Too frequently,
-              get or one of the individual unit getters is used when the
-              function that gave the desired behavior was $(LREF total). This
-              should make it more explicit and help prevent bugs. This function
-              will be removed in June 2015.)
-
-        Returns the number of the given units in this $(D Duration)
-        (minus the larger units).
-
-        $(D d.get!"minutes"()) is equivalent to $(D d.split().minutes).
-      +/
+    // Explicitly undocumented. It will be removed in August 2016. @@@DEPRECATED_2016-08@@@
```

Comment:
Never understood the purpose of not documenting stuff. How would somebody get rid of deprecated thingie if there is no docs?